packages/ingest/src/openai-batches/batchLineCompletion.ts

```diff
@@ -1,30 +1,32 @@
 import { zodResponseFormat } from "openai/helpers/zod";
-import { z } from "zod";
 
-export function getLessonPlanBatchFileLine({
-  lineId,
+export function batchLineCompletion({
+  customId,
+  model,
+  temperature,
   systemPrompt,
   userPrompt,
-  responseSchema,
-  responseSchemaName,
+  responseFormat,
 }: {
-  lineId: string;
+  customId: string;
+  model: string;
+  temperature: number;
   systemPrompt: string;
   userPrompt: string;
-  responseSchema: z.Schema;
-  responseSchemaName: string;
+  responseFormat: ReturnType<typeof zodResponseFormat>;
 }) {
   return {
-    custom_id: lineId,
+    custom_id: customId,
     method: "POST",
     url: "/v1/chat/completions",
```

Comment:
Is this correct? Don't we want the beta endpoint for structured outputs? Or are you not using it here?